.github/copilot-instructions.md

```diff
@@ -12,13 +12,21 @@
 - **Key Dependencies**: Compose Multiplatform, Kobweb Silk, Material Web Components
 - **Current Version**: 0.6.0-SNAPSHOT
 
+<tool_calling>
+You have the capability to call multiple tools in a single response. For maximum efficiency, whenever you need to perform multiple independent operations, ALWAYS invoke all relevant tools simultaneously rather than sequentially. Especially when exploring repository, reading files, viewing directories, validating changes or replying to comments.
+</tool_calling>
+
```

Comment:
This is not available in the `kotlin-common` project. Is this necessary? Why did you add this in this commit that ports the patterns from `kotlin-common`? Why didn't you add this in your first commit?